aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md

```diff
@@ -97,6 +97,12 @@ In the preceding example, `image: hellodockertools` generates the image `hellodo
 
 Prefix the image name with the [Docker Hub](https://hub.docker.com/) username (for example, `dockerhubusername/hellodockertools`) if the image is pushed to the registry. Alternatively, change the image name to include the private registry URL (for example, `privateregistry.domain.com/hellodockertools`) depending on the configuration.
 
+If you want different behavior based on the build configuration (for example, Debug or Release), you can provide that by adding configuration-specific *docker-compose* files. The filenames should be named according to the build configuration (for example, *docker-compose.vs.debug.yml* and *docker-compose.vs.release.yml*) and placed in the same location as the *docker-compose-override.yml* file. 
+
+![Docker compose override files](visual-studio-tools-for-docker/_static/docker-compose-overrides.png)
```

Comment:
I recommend removing this image. The tooling changes too frequently to keep up with the screenshots. The text above describes it well enough.